bridge/engine/EngineStreamDirector.cpp

```diff
@@ -0,0 +1,13 @@
+#include "EngineStreamDirector.h"
+
+namespace bridge
+{
+constexpr size_t EngineStreamDirector::configLadder[6][6] = {
```

Comment:
We should still be able to init these in similar fashion if the ladder contains structs